### DISCUSSION TITLE
Updating EMBA

### DISCUSSION THREAD

**Discussion by Masterchief72** [e-m-b-a/emba#824] [Q&A] [answered]
I currently have v1.2.3 installed and would like to update to v1.3.0.
All documentation that I can find here on GitHub describes a fresh installation, but I cannot find anything about updating.
How do I best proceed ?
Can I just run the installer.sh script again without losing my scan profiles for example ?

**Comment by m-1-k-3** [comment, marked as answer]
If you have installed from a release you probably need a fresh installation. But, this should not be the big thing:
* Download the new release (e.g., v1.3.0)
* extract it to a different directory
* run the installer script
* Alternative: you can also try to just link the external directory to your new install
* Optional: I'm not sure if the docker image will get an automatically update. Probably you need to remove it before the new install
* Finally you can copy your scan profiles to your new installation and delete the old installation

Alternative via Github:
* Use the github version to just do a git pull to get the new stuff
* from time to time check if there is a new docker image available

**Reply by Masterchief72** [reply]
Thanks a lot for the quick reply !
